go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -202,3 +206,44 @@ func TestOuterJoinWithPredicate(t *testing.T) {
 	mcmp.AssertMatchesNoOrder("select A.id1, B.id2 from t1 as A left join t1 as B on A.id1*10 = B.id2 WHERE B.id2 NOT BETWEEN 20 AND 30",
 		`[[INT64(0) INT64(0)] [INT64(1) INT64(10)] [INT64(4) INT64(40)]]`)
 }
+
+// This test ensures that we support PREPARE statement with over 65530 parameters.
```

Comment:
nitpick: Here we say "over 65530 parameters", but then in the test you use exactly that number.

Reply:
Solved via 157ecba666